frontend/include/chpl/framework/all-global-strings.h

```diff
@@ -146,6 +146,8 @@ X(chplBy         , "chpl_by")
 X(chplAlign      , "chpl_align")
 X(chpldocDot     , "chpldoc.")
 X(llvmDot        , "llvm.")
+X(llvmMetadata   , "llvm.metadata")
+X(llvmAssertVectorized, "llvm.assertVectorized")
```

Comment:
I am personally not a big fan of using whitespaces to create "tables" in code, but that's what we have and you should adjust this "table". (And that's why I am not a big fan)